src/Builder/DatagridBuilder.php

```diff
@@ -156,14 +156,15 @@ public function getBaseDatagrid(AdminInterface $admin, array $values = []): Data
         if (!$query instanceof ProxyQueryInterface) {
             throw new \TypeError(sprintf('The admin query MUST implement %s.', ProxyQueryInterface::class));
         }
+        /** @phpstan-var ProxyQueryInterface<object> $query */
```

Comment:
Is this one necessary? Isnt the admin->createQuery returning already with the generic applied?

Reply:
It's considered as a 
```
Sonata\AdminBundle\Datagrid\ProxyQueryInterface&Sonata\DoctrineORMAdminBundle\Datagrid\ProxyQueryInterface<object>
```

And I have the error:
```
Method Sonata\DoctrineORMAdminBundle\Builder\DatagridBuilder::getBaseDatagrid() should return                                  
         Sonata\AdminBundle\Datagrid\DatagridInterface<Sonata\DoctrineORMAdminBundle\Datagrid\ProxyQueryInterface<object>> but returns  
         Sonata\AdminBundle\Datagrid\Datagrid<Sonata\DoctrineORMAdminBundle\Datagrid\ProxyQueryInterface>.  
```